gobblin-data-management/src/main/java/gobblin/data/management/copy/hive/HiveCopyEntityHelper.java

```diff
@@ -361,6 +362,11 @@
         if (HiveUtils.isPartitioned(this.dataset.table)) {
           this.sourcePartitions = HiveUtils.getPartitionsMap(multiClient.getClient(source_client), this.dataset.table,
               this.partitionFilter, this.hivePartitionExtendedFilter);
+          for (Map.Entry<List<String>, Partition> partition : this.sourcePartitions.entrySet()) {
```

Comment:
Please don't put this code here. `HiveCopyEntityHelper` is a generic class that should not have anything avro specific. You can look at `HiveAvroCopyEntityHelper` to see how this kind of modification was done before.